lib/favicons.php

```diff
@@ -9,7 +9,7 @@
 function download_favicon($website, $dest) {
 	global $favicons_dir, $default_favicon;
 
-	syslog(LOG_DEBUG, 'FreshRSS Favicon discovery GET ' . $website);
+	syslog(LOG_INFO, 'FreshRSS Favicon discovery GET ' . $website);
```

Comment:
We should indeed improve the logging options. I think it is important to have logs for the outgoing queries.